src/Piece/PieceCreate/PieceCreate.test.js

```diff
@@ -0,0 +1,99 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import { render } from '@folio/jest-config-stripes/testing-library/react';
+import {
+  ORDER_FORMATS,
+  useOrderLine,
+} from '@folio/stripes-acq-components';
+
+import { useTitle } from '../../common/hooks';
+import { PieceFormContainer } from '../PieceForm';
+import { PieceCreate } from './PieceCreate';
+
+jest.mock('@folio/stripes-acq-components', () => {
+  return {
```

Comment:
we can use the same formating https://github.com/folio-org/ui-receiving/pull/554/files#diff-34d324df8e9e047569ed1a4aed932471f36691e5b7c0041a389615e675c36006R20-R21